ws_fnc/AI/fn_enterBuilding.sqf

```diff
@@ -35,19 +35,19 @@ while {count _units > 0 && count _barray > 0} do {
 
 		// Get variables to check if building can be used
 		_bUnits = _building getVariable ["ws_bUnits",0];
-		_bposleft = _building getVariable ["ws_bPosLeft",_bposarray];
+		_bposleft = _building getVariable ["ws_bPosLeft",+_bposarray];
 
 		// Loop until we find a good building
 		while {count _barray > 0 && {count _bposLeft == 0 || (_bUnits / (count _bposarray) >= _threshold)}} do {
 			_barray = _barray - [_building];
 
 			_building = selectRandom _barray;
 			_bposarray = _building getVariable ["ws_bPos",[]];
-			if (count _bposleft == 0) then {_bposarray = [_building] call ws_fnc_getBpos;};
```

Comment:
Here `_bposleft` was used, but never set (for the current building).
So I changed it to `_bposarray`